.github/workflows/build-macos.yml

```diff
@@ -43,6 +43,10 @@ jobs:
         uses: actions/checkout@v3
       - name: Clean up Mono
         run: sudo ./.github/workflows/mono-uninstall.sh
+      - name: Setup - Xcode
+        if: matrix.os == 'macos-11'
+        run: |
+          sudo xcode-select -s /Applications/Xcode_11.7.app
```

Comment:
If XCode 13.2.1 is available on OSX 11 then I'd prefer to use and require that, unless there's some reason not to...

Reply:
If we don't need to test minimum possible XCode then `Setup - Xcode` step could be removed.